src/openms/source/CONCEPT/Constants.cpp

```diff
@@ -154,6 +154,9 @@ namespace OpenMS
 
     namespace UserParam
     {
+      // User parameter name for the M/Z of other chromatograms which have been merged into this one
+      const std::string MERGED_WITH = "merged_with";
```

Comment:
this string constant is probably too general for the narrow task it is used for and there might be some potential name clashes (i.e. it's not that unrealistic that a user already stored a merged_with meta value with different meaning).
So maybe something like "merged_chromatogram_mzs" ?